guides/source/form_helpers.md

```diff
@@ -1025,8 +1119,11 @@ The following form allows a user to create a `Person` and its associated address
 <% end %>
 ```
 
+When an association accepts nested attributes `fields_for` renders its block once for every element of the association. In particular, if a person has no addresses it renders nothing.
```

Comment:
Some punctuation updates: When an association accepts nested attributes, fields_for renders its block once for every element of the association. In particular, if a person has no addresses, it renders nothing.